### ISSUE TITLE
Proper set name truncation

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#355]
Set names are just clipped when overflowing, but it would be good to have proper ellipsis truncation

**Comment by NateLanza** [comment]
@JakeWags isn't this completed?

**Comment by JakeWags** [comment]
@NateLanza No, the current behavior just clips the word rather than appending ellipses. See "shortness of breath" being "shortness of "

![Image](https://github.com/user-attachments/assets/4b3d90ec-d1f1-479a-a875-db00323b2289)
